### PR TITLE
Feature/ins 401 details of component methods

### PR DESCRIPTION
**Pull request by vierrra**

### CODE REVIEW

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+        configureNotApplicableView()
+        configureNotApplicableLabel()
+        configureNotApplicableSwitch()
+        configureLowerNotApplicableViewSeparator()
+        configureApprovedButton()
+        configureReprovedButton()
+        configureInspectionLabel()
+    }
+    
+    fileprivate func configureView() {
```
Comment (Leandro97) [inline StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Why are you creating this view? What is its purpose?

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+    public func makeShadow(button: UIButton) {
+        button.layer.shadowPath    = UIBezierPath(rect: button.bounds).cgPath
+        button.layer.shadowRadius  = 4
+        button.layer.shadowOffset  = CGSize(width: 0, height: 3)
+        button.layer.shadowOpacity = 0.1
+    }
+}
+
+@objc public protocol VerifiedMethodDetailViewDelegate {
+    @objc func goToAddApproveInspectionData()
```
Comment (Leandro97) [inline StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Change the name of this function. It should be goToApproveVerifiedMethod

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+    @objc func goToAddReproveInspectionData()
```
Comment (Leandro97) [inline StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Change the name of this function. It should be goToReproveVerifiedMethod

File: StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+//  Created by Renato Vieira on 7/7/20.
+//  Copyright © 2020 Stant. All rights reserved.
+//
+
+import UIKit
+import StantUiIosLibrary
+
+class VerifiedMethodDetailViewController: UIViewController {
+    public var verifiedMethodDetailView: VerifiedMethodDetailView?
+    public var methodItemDescription: String = "Observar a eliminação de qualquer foco de umidade de modo que a superfície apresente-se seca quando da execução da pintura"
```
Comment (Leandro97) [inline StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift]: Indentation

File: StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    
+    let navigationBarHeight: CGFloat = 60
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (Leandro97) [inline StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift]: Remove extra line

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift (hunk 2 above)
Comment (vierrra) [reply StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Done

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift (hunk 3 above)
Comment (vierrra) [reply StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Done

File: StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift (hunk 4 above)
Comment (vierrra) [reply StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift]: Done

File: StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift (hunk 5 above)
Comment (vierrra) [reply StantUiIosLibraryDemo/StantUiIosLibraryDemo/DemoApplication/ItemMethod/VerifiedMethodDetailViewController.swift]: done

File: StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift (hunk 1 above)
Comment (vierrra) [reply StantUiIosLibrary/Classes/Cards/MethodDetail/VerifiedMethodDetailCard.swift]: Removed

**Review by Leandro97 — APPROVED** [review]

**Review by andersonfernandes — APPROVED** [review]

**Pull request merged**